eng/performance/scenarios.proj

```diff
@@ -45,8 +45,12 @@
     <Scenario Include="New VB Console Template">
       <PayloadDirectory>$(ScenariosDir)emptyvbconsoletemplate</PayloadDirectory>
     </Scenario>
+    <Scenario Include="WinForms Large">
+      <PayloadDirectory>$(ScenariosDir)windowsformslarge</PayloadDirectory>
+    </Scenario>
```

Comment:
Should this be conditioned on the run occurring on windows? At the very least the startup test should be as I don't think winforms runs on Linux.